lib/logster/logger.rb

```diff
@@ -55,8 +51,7 @@ def add(*args, &block)
     end
 
     def level
-      ol = @override_levels
-      (ol && ol[Thread.current.object_id]) || @level
+      Thread.current[@logster_override_level_key] || @level
```

Comment:
```suggestion
      override_level || @level
```

Reply:
hmmm I think carrying the micro optimisation of erasing one method call is ok, probably does not matter at all for truffle and long term will not matter for ruby, it is a very hot method cause it is called super frequently

Reply:
:+1: Understood :)